README.md

```diff
@@ -1,6 +1,6 @@
 # CDPLogger Client for JavaScript
 
-A simple JavaScript interface for communicating with CDP applications that include a CDPLogger component to retrieve historic data.
+A simple JavaScript interface for communicating with CDP applications that include a CDPLogger component to retrieve historic data. For more information about CDP Studio see https://cdpstudio.com/.
```

Comment:
Keep 120 char line lenght.

Also, I think there should be a comma before "see"  `... CDP Studio, see https://cdpstudio.com/.`